linux_os/guide/system/software/sap/accounts_authorized_local_users_sidadm_orasid.rule

```diff
@@ -19,9 +19,8 @@ description: |-
     OVAL regular expression is used for the user list.
     <br /> <br />
     Test result of both <tt>fail</tt> or <tt>error</tt> means mismatch of user names and
-    SAP system. Nevertheless, the bash remediation commands can always be used to delete
-    unexpected users on the operating system. It works in all cases zero, one and multiple
-    SAP systems on the operating system (or virtual machine).
+    SAP system.<tt>*</tt> The bash remediation commands can be used to delete unexpected users on
```

Comment:
What do you mean by the <tt>*</tt> ? It feels out of place there. Was it supposed to be some pointer to the warning section?

Reply:
@dahaic yes, it's supposed to be notation that there is a warning section for  this sentence. Any suggestion?